app/controllers/admin/collections_controller.rb

```diff
@@ -4,6 +4,7 @@ class Admin::CollectionsController < AdminController
   # GET /collections
   # GET /collections.json
   def index
+    authorize! :read, Kithe::Model
```

Comment:
You are technically right: the AdminController does check on *all* actions that the user can `access_staff_functions`, so this line doesn't accomplish anything for our current access policy.

It still feels more elegant and future-proof to me to have an authorize! check at the beginning of `:read` methods in these controllers: it maintains a clean separation between `:access_staff_functions` in general and "interacting with collection objects" in particular. 

Another way of looking at this: when you browse through the code 10 months from now, you won't have the extra cognitive load of having to remind yourself why some controllers have an authorize! and why some don't.

Reply:
That makes a lot of sense, but counter-argument:  

What does this check actually mean? I think it means that the only people who can see the list of Collections are people who have the right to see _all_ Collections, published and unpublished. 

I'm not sure that's the right check for the :index action though -- if for some reason you only had the right to see published collections but still had `: access_staff_functions`, you could probably see Collection#index, which would filter out unpublished collections?

So if you want to duplicate the check, is the right check `can? :read, Kithe::model`, or `can? :access_staff_functions`?

Reply:
Honestly part of it here is that I find `can? :read, Kithe::Model` check one of the more confusing ones  -- wait, what's a Kithe::Model? And what does this even mean?

I was kind of hoping it's use would end up restricted _only_ to the `can_see_unpublished_records?` cover method we added, since that documetns what the heck it is.  So I'm unthrilled to see it's use proliferating, especially for hypothetical just-for-consistency unneeded purposes.

Reply:
Point taken. I'm getting rid of this line; using a comment instead.